src/App.js

```diff
@@ -11,44 +26,52 @@ class App extends React.Component {
       currWord: getRandomWord(),
       // guessedLetters stores all letters a user has guessed so far
       guessedLetters: [],
-      // Insert num guesses left state here
-      // Insert form input state here
+      numGuess: 10,
+      userWord: "",
+      result: "",
+      score: 0,
     };
   }
 
-  generateWordDisplay = () => {
-    const wordDisplay = [];
-    // for...of is a string and array iterator that does not use index
-    for (let letter of this.state.currWord) {
-      if (this.state.guessedLetters.includes(letter)) {
-        wordDisplay.push(letter);
-      } else {
-        wordDisplay.push("_");
-      }
-    }
-    return wordDisplay.toString();
+  reset = () => {
+    this.setState({
+      currWord: getRandomWord(),
+      guessedLetters: [],
+      numGuess: 10,
+      result: "",
+    });
```

Comment:
You could create a static variable outside of the component for this. 
```suggestion
    const defaultState = {
      currWord: getRandomWord(),
      guessedLetters: [],
      numGuess: 10,
      result: "",
      score: 0,
    };
```

and then reuse it whenever necessary